modules/402-ingress-nginx/templates/controller/controller.yaml

```diff
@@ -127,6 +127,9 @@ spec:
       annotations:
         traffic.sidecar.istio.io/includeInboundPorts: ""
         traffic.sidecar.istio.io/includeOutboundIPRanges: {{ $context.Values.global.discovery.serviceSubnet | quote }}
+      {{- if $crd.spec.enableIstioSidecar }}
+        inject.istio.io/templates: sidecar,d8-hold-istio-proxy-termination-until-application-stops
+      {{- end }}
```

Comment:
```suggestion
        inject.istio.io/templates: sidecar,d8-hold-istio-proxy-termination-until-application-stops
```

Reply:
There is already       {{- if $crd.spec.enableIstioSidecar }} several lines above.

Reply:
![image](https://github.com/deckhouse/deckhouse/assets/41360396/db8bf97f-5a6e-4841-a47f-a7ec696fe9e2)
doesn't look like Label section at all

Reply:
Yeah. It's an annotation. It's supposed to be.

Reply:
and why would we need having `$crd.spec.enableIstioSidecar` checked twice, before `annotations:` section and inside it?